kolibri/core/package.json

```diff
@@ -21,7 +21,8 @@
     "js-cookie": "^3.0.5",
     "knuth-shuffle-seeded": "^1.0.6",
     "kolibri-constants": "0.2.6",
-    "kolibri-design-system": "5.0.0-rc2",
+    "kolibri-design-system": "https://github.com/BabyElias/kolibri-design-system#7764ef4b5f9b66ebd90d2ad389ae3d44252437a0",
```

Comment:
Before merging this PR, we will replace this by the newest KDS release that will contain https://github.com/learningequality/kolibri-design-system/pull/727